docs/quickstart.md

```diff
@@ -79,29 +63,27 @@ Authenticate your CLI with Fiberplane
 fp login
 ```
 
-To register a proxy run a command `fp proxies create`:
+To register a daemon run a command `fp daemon create`:
 
+<!-- markdownlint-disable -->
 ```shell
-$ fp proxies create my-proxy-name
-				Name:  my-proxy-name
-          ID:  <generated_proxy_id>
+$ fp daemon create my-daemon-name
+		Name:  my-daemon-name
+          ID:  <generated_daemon_id>
       Status:  disconnected
  Datasources:  (none)
-        Token  <YOUR_PROXY_API_TOKEN> <-- SAVE THIS FOR LATER!
+        Token  <YOUR_DAEMON_API_TOKEN> <-- SAVE THIS FOR LATER!
 ```
+<!-- markdownlint-enable -->
 
-You can leave out the proxy name to have one randomly generated, but we
-recommend naming it according to the environment it will be deployed to (for
-example, `production`). Note that proxy names must follow the [Fiberplane name
-format](doc:configuration-help-faq#resource-names).
+You can leave out the proxy name to have one randomly generated, but we recommend naming it according to the environment it will be deployed to (for example, `production`). Note that proxy names must follow the [Fiberplane name format](doc:configuration-help-faq#resource-names).
```

Comment:
```suggestion
You can leave out the daemon name to have one randomly generated, but we recommend naming it according to the environment it will be deployed to (for example, `production`). Note that daemon names must follow the [Fiberplane name format](doc:configuration-help-faq#resource-names).
```